src/main/resources/alma/fix/describedBy.fix

```diff
@@ -29,9 +29,12 @@ set_array("describedBy.type[]", "BibliographicDescription")
 # 		<call-macro name="simplify-date" dname="dateModified" dsource="MNG  .d"/>
 copy_field("MNG  .b","describedBy.dateCreated")
 copy_field("MNG  .d","describedBy.dateModified")
-replace_all("describedBy.dateCreated|dateModified","-","")
-replace_all("describedBy.dateCreated|dateModified"," .*","")
-replace_all("describedBy.dateCreated|dateModified","c|©|\\s?|,|.|:|;|/|=","")
```

Comment:
Any other potential bugs notwithstanding, I would strongly advise against relying on this wrong behaviour in the current Metafix implementation! See metafacture/metafacture-fix#143.